src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -455,6 +455,14 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     refetchInterval: 60_000,
   })
 
+  const poolMissing = useMemo(() => {
+    if (poolAsset === undefined || pools === undefined) {
+      return false
```

Comment:
Should this return `true` here for paranoia if we don't have a `poolAsset` or `pools`?

Reply:
Seconding this, but only if it is preceeded by loading checks that return false when loading pool data